recipes/z3-solver/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "z3-solver" %}
+{% set version = "4.9.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/z3-solver-{{ version }}.tar.gz
+  sha256: 5ed51c9748a23b6e21af9a526fa21f7ff0a76fd696aff436676300f162e10ddc
+  patches:
+    - 0001-target_os_osx.patch  # [osx]
+    - 0002-CMAKE_CXX_COMPILER-win.patch  # [win]
+
+build:
+  script:
+    - set "CMAKE_GENERATOR=Ninja"  # [win]
```

Comment:
```suggestion
        - del /q /f pyproject.toml                       # [win]
        - set "CMAKE_GENERATOR=Ninja"  # [win]
```
Can we try this?

Reply:
I'm somewhat worried that somebody is pulling in https://pypi.org/project/cmake/